cmd/geth/main.go

```diff
@@ -510,6 +511,9 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend) {
 			threads = ctx.GlobalInt(utils.LegacyMinerThreadsFlag.Name)
 			log.Warn("The flag --minerthreads is deprecated and will be removed in the future, please use --miner.threads")
 		}
+		if threads == 0 && ctx.GlobalBool(utils.DeveloperFlag.Name) {
```

Comment:
Instead of checking `threads == 0`, maybe should check `!ctx.GlobalIsSet(utils.MinerThreadsFlag.Name) && !ctx.GlobalIsSet(utils.LegacyMinerThreadsFlag.Name)`. Think so?

Reply:
If you check from line 509 till here it covers flag check, so I thought it will be cleaner this way. What do you think? At the end what we want is to deal with `threads=0`, on the other hand this will not allow user forcing `--miner.threads=0`

Reply:
I would prefer to allow users the full extent of potential control of the system, even at the cost of a little code cleanliness.  So I think we should let them be able to set `miner.threads=0`. Come to think of it, can't `miner.threads=0` be used to turn off CPU mining while still allowing GPU mining via `eth_getWork`? (See https://github.com/etclabscore/core-geth/issues/207)